packages/llvm/llvm.16.0.6+nnp/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "The OCaml bindings distributed with LLVM"
+description: "Note: LLVM should be installed first."
+maintainer: "Alan Hu <alanh@ccs.neu.edu>"
+authors: [
+  "Alan Hu <alanh@ccs.neu.edu>"
+  "Kate <kit-ty-kate@outlook.com>"
+  "Gordon Henriksen <gordonhenriksen@mac.com>"
+  "whitequark <whitequark@whitequark.org>"
+  "The LLVM team"
+]
+license: "Apache-2.0 WITH LLVM-exception"
+homepage: "http://llvm.moe"
+doc: "http://llvm.moe/ocaml"
+bug-reports: "http://llvm.org/bugs/"
+depends: [
+  "ocaml" {>= "4.00.0"}
+  "dune" {>= "2.7"}
+  "ctypes" {>= "0.4"}
+  "conf-llvm" {build & = "16"}
```

Comment:
```suggestion
  "conf-llvm" {build & = "16"}
  "conf-zstd"
```
I hope this would resolve
```
# File "src/backends/AArch64/static/dune", line 2, characters 0-1217:
#  2 | (library
#  3 |  (name llvm_AArch64_static)
#  4 |  (public_name llvm_AArch64.static)
#  5 |  (implements llvm_AArch64)
#  6 |  (foreign_stubs
#  7 |   (language c)
#  8 |   (names AArch64_ocaml)
#  9 |   (extra_deps llvm_ocaml.h)
# 10 |   (flags (-I/opt/homebrew/Cellar/llvm/16.0.6/include  -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS "-DTARGET=AArch64")))
# 11 |  (c_library_flags (-L/opt/homebrew/Cellar/llvm/16.0.6/lib -Wl,-search_paths_first -Wl,-headerpad_max_install_names -lstdc++ -lLLVMAArch64Disassembler -lLLVMMCDisassembler -lLLVMAArch64AsmParser -lLLVMAArch64CodeGen -lLLVMCFGuard -lLLVMGlobalISel -lLLVMSelectionDAG -lLLVMAsmPrinter -lLLVMCodeGen -lLLVMTarget -lLLVMScalarOpts -lLLVMInstCombine -lLLVMAggressiveInstCombine -lLLVMObjCARCOpts -lLLVMTransformUtils -lLLVMBitWriter -lLLVMAnalysis -lLLVMProfileData -lLLVMSymbolize -lLLVMDebugInfoPDB -lLLVMDebugInfoMSF -lLLVMDebugInfoDWARF -lLLVMObject -lLLVMTextAPI -lLLVMMCParser -lLLVMIRReader -lLLVMAsmParser -lLLVMBitReader -lLLVMAArch64Desc -lLLVMAArch64Utils -lLLVMCore -lLLVMRemarks -lLLVMBitstreamReader -lLLVMAArch64Info -lLLVMMC -lLLVMDebugInfoCodeView -lLLVMBinaryFormat -lLLVMTargetParser -lLLVMSupport -lLLVMDemangle
# 12 | -lm /opt/homebrew/lib/libz3.dylib -lz -lzstd -lcurses -lxml2)))
# (cd _build/default && /Users/mac1000/.opam/4.14.1/bin/ocamlmklib.opt -g -o src/backends/AArch64/static/llvm_AArch64_static_stubs src/backends/AArch64/static/AArch64_ocaml.o -ldopt -L/opt/homebrew/Cellar/llvm/16.0.6/lib -ldopt -Wl,-search_paths_first -ldopt -Wl,-headerpad_max_install_names -ldopt -lstdc++ -ldopt -lLLVMAArch64Disassembler -ldopt -lLLVMMCDisassembler -ldopt -lLLVMAArch64AsmParser -ldopt -lLLVMAArch64CodeGen -ldopt -lLLVMCFGuard -ldopt -lLLVMGlobalISel -ldopt -lLLVMSelectionDAG -ldopt -lLLVMAsmPrinter -ldopt -lLLVMCodeGen -ldopt -lLLVMTarget -ldopt -lLLVMScalarOpts -ldopt -lLLVMInstCombine -ldopt -lLLVMAggressiveInstCombine -ldopt -lLLVMObjCARCOpts -ldopt -lLLVMTransformUtils -ldopt -lLLVMBitWriter -ldopt -lLLVMAnalysis -ldopt -lLLVMProfileData -ldopt -lLLVMSymbolize -ldopt -lLLVMDebugInfoPDB -ldopt -lLLVMDebugInfoMSF -ldopt -lLLVMDebugInfoDWARF -ldopt -lLLVMObject -ldopt -lLLVMTextAPI -ldopt -lLLVMMCParser -ldopt -lLLVMIRReader -ldopt -lLLVMAsmParser -ldopt -lLLVMBitReader -ldopt -lLLVMAArch64Desc -ldopt -lLLVMAArch64Utils -ldopt -lLLVMCore -ldopt -lLLVMRemarks -ldopt -lLLVMBitstreamReader -ldopt -lLLVMAArch64Info -ldopt -lLLVMMC -ldopt -lLLVMDebugInfoCodeView -ldopt -lLLVMBinaryFormat -ldopt -lLLVMTargetParser -ldopt -lLLVMSupport -ldopt -lLLVMDemangle -ldopt -lm -ldopt /opt/homebrew/lib/libz3.dylib -ldopt -lz -ldopt -lzstd -ldopt -lcurses -ldopt -lxml2)
# ld: library not found for -lzstd
# clang: error: linker command failed with exit code 1 (use -v to see invocation)
```